### PR TITLE
Restyling

### PR DESCRIPTION
**Pull request by bluepichu**
Changes include:
- Fixed "About" Page
- Fixed "Contacts" Page
- Fixed Author Pages
- Better Syntax Highlighting (including language labeling and copy system)
- Restyled Headers
- Header Classes

### CODE REVIEW

File: client/scss/vars.scss [hunk 1]
```diff
@@ -27,4 +27,9 @@ $wire-width: 8px;
 
 
 
-$width-small: 125ch;
\ No newline at end of file
+$width-small: 125ch;
```
Comment (zwade) [inline client/scss/vars.scss]: very useful change right here

File: server/stachehelper.js [hunk 2]
```diff
@@ -6,24 +6,91 @@ var moment      = require("moment")
...
+}
 
 marked.setOptions({
 	gfm: true,
 	highlight: function(code, lang){
-		return highlight.highlight(code, highlight.getLanguage(lang ? lang : "text"))
+		if (lang !== undefined) {
+			let highlighted = highlight.highlight(lang, code).value
+
+			if (lang === "bash") {
```
Comment (zwade) [inline server/stachehelper.js:29]: I dun't like dis. Is it possible to specify a new bash highlight script?

**Comment by zwade** [comment]
k

File: server/stachehelper.js (hunk 2 above)
Comment (bluepichu) [inline server/stachehelper.js:29]: Not without modifying highlight.js source, which would get overwritten every time we update it.

**Pull request merged**